Selection_Sort/Selection_Sort.php

```diff
@@ -0,0 +1,36 @@
+<?php
+    //Selection Sort function 
+    function selection_sort($ar,$size)
+    {
+        for ($i = 0; $i < $size-1; $i=$i+1)  
+        {  
+            // Find the minimum element in unsorted array  
+            $min_idx = $i;  
+            for ($j = $i+1; $j < $size; $j=$j+1)  
+            if ($ar[$j] < $ar[$min_idx])  
+                $min_idx = $j;  
+ 
+            // Swap the minimum element with the first element of unsorted array 
+            $temp=$ar[$min_idx];
+            $ar[$min_idx]=$ar[$i];
+            $ar[$i]=$temp;  
+        }  
+
+        // Print the sorted array
+        for($i=0;$i<$size;$i=$i+1)
+            echo "$ar[$i] ";
+    }
+
+// Sample Input 
```

Comment:
Indent this as well.
It should align with upper block.

Reply:
Do you mean the for loop to print array? shouldn't that be indented with the same spaces as the i loop above (line 5) only?